### PR TITLE
Add docs for snapshots

### PR DESCRIPTION
**Pull request by lamminade**
#### Changes made
- Add snapshot tab to index
- Add list and get snapshot doc

#### Related PRs
- [GCP PR](https://github.com/cloudops/cloudmc-gcp-plugin/pull/262)

### CODE REVIEW

File: source/includes/gcp/_snapshots.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+`name` <br/>*string* | Name of the resource; provided by the client when the resource is created. 
+
+<!-------------------- RETRIEVE A SNAPSHOT -------------------->
+
+#### Retrieve a snapshot
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/instances/7059283128121212212"
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_snapshots.md]: `instances` -> `snapshots`

File: source/includes/gcp/_snapshots.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+<code>
+   GET /service/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/snapshots
+</code>
+
+Retrieve a list of all snapshots.
+
+Attributes | &nbsp;
+------- | -----------
+`description` <br/>*string* | The snapshot description
```
Comment (patrickspensieri) [inline source/includes/gcp/_snapshots.md]: Just missing a period here and on description for retrieve snapshot by id

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/gcp/_snapshots.md (hunk 2 above)
Comment (lamminade) [reply source/includes/gcp/_snapshots.md]: [547f14e](https://github.com/cloudops/cloudmc-api-docs/pull/178/commits/547f14e3f03c4e0d8b89f110440893d187a778b1)

File: source/includes/gcp/_snapshots.md (hunk 1 above)
Comment (lamminade) [reply source/includes/gcp/_snapshots.md]: [547f14e](https://github.com/cloudops/cloudmc-api-docs/pull/178/commits/547f14e3f03c4e0d8b89f110440893d187a778b1)

**Review by GuillaumeVialle — APPROVED** [review]

**Pull request merged**